video/out/vo_gpu_next.c

```diff
@@ -688,6 +692,7 @@ static bool map_frame(pl_gpu gpu, pl_tex *tex, const struct pl_source_frame *src
         frame->acquire = hwdec_acquire;
         frame->release = hwdec_release;
         frame->num_planes = desc.num_planes;
+        frame->repr.bits = plane_bits_from_imgfmt(par.imgfmt);
```

Comment:
Wouldn't this be enough?

```suggestion
        frame->repr.bits = mpi->params.repr.bits;
```

Reply:
Yes, I think you're right. Do you think it's still worth keeping the cosmetic change to `plane_bits_from_imgfmt` (being split off from the already complicated `plane_data_from_imgfmt`)?

Reply:
Decided to drop the refactor for now, the next person to touch this function can remove that logic